exchanges/poloniex/poloniex.go

```diff
@@ -877,6 +882,10 @@ func (p *Poloniex) SendAuthenticatedHTTPRequest(ep exchange.URL, method, endpoin
 		Verbose:       p.Verbose,
 		HTTPDebugging: p.HTTPDebugging,
 		HTTPRecording: p.HTTPRecording,
+	}
+
+	return p.SendPayload(context.Background(), request.Unset, func() (*request.Item, error) {
+		return item, nil
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Poloniex has rate limits